spec/helpers/providers_helper_spec.rb

```diff
@@ -112,5 +136,31 @@
         expect(url_for_application(legal_aid_application)).to eq("/providers/applications/#{legal_aid_application.id}/means/remove_dependants/#{dependant.id}?locale=en")
       end
     end
+
+    context "when the application predates the 2023 surname at birth issue" do
+      let(:legal_aid_application) do
+        travel_to Date.parse("2023-12-25") do
+          create(:legal_aid_application, :with_multiple_proceedings_inc_section8, provider_step:)
+        end
+      end
+
+      context "and the provider step should expire it" do
```

Comment:
Sorry if this is too picky, but I don't think it's clear to me what 'should expire it' means here. Same for the next test (line 156)